qabelbox/src/main/java/de/qabel/qabelbox/activities/MainActivity.java

```diff
@@ -228,6 +230,16 @@ private boolean uploadUri(Uri uri, String targetFolder) {
     protected void onCreate(Bundle savedInstanceState) {
 
         super.onCreate(savedInstanceState);
+        LocalQabelService service = QabelBoxApplication.getInstance().getService();
```

Comment:
if we set the service to private so its run on the same process as the rest of the application